etc/org.ops4j.pax.logging.cfg

```diff
@@ -1,30 +1,61 @@
-# Root logger configuration.  This defines the default values to be used.  They may be overwritten by more specific
-# settings below.
-log4j.rootLogger=WARN, out, osgi:*
-log4j.throwableRenderer=org.apache.log4j.OsgiThrowableRenderer
+# Colors for log level rendering
+color.fatal = bright red
+color.error = bright red
+color.warn = bright yellow
+color.info = bright green
+color.debug = cyan
+color.trace = cyan
+
+# Common pattern layout for appenders
+log4j2.pattern = %d{ISO8601} | %-5p | %-16t | %-32c{1} | %X{bundle.id} - %X{bundle.name} - %X{bundle.version} | %m%n
```

Comment:
This pattern generates insanely long lines.  With a quick test run I generated lines of 732 characters.  Any chance we can remove the `%-16t` and `%X{bundle.id} - %X{bundle.name} - %X{bundle.version}` fields?  This tells us (relatively) useless information like `fileinstall-/home/greg/opencast/upstream/build/opencast-dist-develop-7-SNAPSHOT/etc/encoding` and `64 - opencast-composer-ffmpeg - 7.0.0.SNAPSHOT`

Reply:
I agree. I've replaced this with the currently used pattern.